features/filesystem/littlefs/.travis.yml

```diff
@@ -0,0 +1,70 @@
+script:
```

Comment:
features/filesystem/littlefs/.travis.yml - is this just copy-paste from littlefs repo and works with travis within this repository?

Reply:
Unfortunately, Travis doesn't support delegating testing to other .travis.ymls. I need to move all of these tests into the central .travis.yml.

@studavekar, @bridadan, unless you guys know any clever outs?

Reply:
I'm not aware of any mechanism like that off the top of my head (doesn't mean it doesn't exist, just needs a google).

But I'm not sure if you actually want to do that, since this is currently cloning the mbed-os repo. The repo will already by cloned and properly (pre)merged, so you'll want to work within that repo if possible. I think you'll need to do the integrating. Good news is these share a decent amount of requirements, so that work will hopefully be lighter.